locations/spiders/yettel_bg.py

```diff
@@ -1,8 +1,7 @@
-import io
-
-from openpyxl import load_workbook
```

Comment:
Can you check to see if this library is used elsewhere?

Reply:
It's currently being used in `dbe_gov_schools_za.py` and `dbe_gov_ecd_za.py`.